assets/helpers/tracking/guTracking.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+// ----- Campaigns ----- //
+
+const campaigns : {
+  [string]: string[],
+} = {
+  baseline_test: [
+    'gdnwb_copts_memco_sandc_support_baseline_support_banner',
```

Comment:
Maybe extract 'gdnwb_copts_memco_sandc_support_baseline_support_' out into a const and then append the final part of the code onto it, for readability and ease of maintenance

Reply:
Do you mean something like this?

```js
const prefix = 'gdnwb_copts_memco_sandc_support_baseline_support_';

const campaigns : {
  [string]: string[],
} = {
  baseline_test: [
    'banner',
    'epic',
    'liveblog',
    'header_become_supporter',
    'header_subscribe',
    'side_menu_become_supporter',
    'side_menu_subscribe',
  ].map(suffix => `${prefix}${suffix}`)
};
```

Reply:
Yes, nice.

Reply:
Updated to use @JustinPinner's solution, by popular vote, as discussed online elsewhere (as opposed to offline).